src/chainparams.cpp

```diff
@@ -196,16 +196,22 @@ class CMainParams : public CChainParams {
             ( 1750000, uint256S("0x5c8e7327984f0d6f59447d89d143e5f6eafc524c82ad95d176c5cec082ae2001"))
             ( 2000000, uint256S("0x9914f0e82e39bbf21950792e8816620d71b9965bdbbc14e72a95e3ab9618fea8"))
             ( 2031142, uint256S("0x893297d89afb7599a3c571ca31a3b80e8353f4cf39872400ad0f57d26c4c5d42"))
+            ( 2250000, uint256S("0x0a87a8d4e40dca52763f93812a288741806380cd569537039ee927045c6bc338"))
             ( 2510150, uint256S("0x77e3f4a4bcb4a2c15e8015525e3d15b466f6c022f6ca82698f329edef7d9777e"))
+            ( 2750000, uint256S("0xd4f8abb835930d3c4f92ca718aaa09bef545076bd872354e0b2b85deefacf2e3"))
+            ( 3000000, uint256S("0x195a83b091fb3ee7ecb56f2e63d01709293f57f971ccf373d93890c8dc1033db"))
+            ( 3250000, uint256S("0x7f3e28bf9e309c4b57a4b70aa64d3b2ea5250ae797af84976ddc420d49684034"))
+            ( 3500000, uint256S("0xeaa303b93c1c64d2b3a2cdcf6ccf21b10cc36626965cc2619661e8e1879abdfb"))
+            ( 3606083, uint256S("0x954c7c66dee51f0a3fb1edb26200b735f5275fe54d9505c76ebd2bcabac36f1e"))
         };
 
         chainTxData = ChainTxData{
-            // Data as of block 77e3f4a4bcb4a2c15e8015525e3d15b466f6c022f6ca82698f329edef7d9777e (height 2510150).
-            // Tx estimate based on average of year 2018 (~27k transactions per day)
-            1544484077, // * UNIX timestamp of last checkpoint block
-            42797508,   // * total number of transactions between genesis and last checkpoint
+            // Data as of block 954c7c66dee51f0a3fb1edb26200b735f5275fe54d9505c76ebd2bcabac36f1e (height 3606083).
+            // Tx estimate based on average of year 2021 (~40k transactions per day)
```

Comment:
2021 has seen some above average activity I'd guess. Are we expecting this trend to hold on for a while longer?

Reply:
2020 doesn't seem much slower, actually. Year start/end looks like:

```
UpdateTip: new best=f6bd77f8c2ba6053bea94120a24f41a8480a38954098af5ab5e0f4d411ccfea8 height=3043797 version=0x00620103 log2_work=73.594646 tx=54490003 date='2020-01-01 00:00:23' progress=0.831138 cache=245.8MiB(268360tx)
UpdateTip: new best=cf8997cac1ad13837a821b2c02fadb07b39500f8f1a13ed01552ac3c0b90c1a3 height=3606004 version=0x00620104 log2_work=74.195891 tx=71033379 date='2021-02-13 10:47:18' progress=1.000000 cache=6.3MiB(1744tx)
```

71,033,379 - 54,490,003 = 16,543,376
16,543,376 / 365 / 24 / 3,600 = 0.5246 tx/s

Thoughts?

Reply:
I'm good with 0.46 as an estimate.

Reply:
Works for me